pkg/postgres/query/compose.go

```diff
@@ -37,6 +37,40 @@ func composeQuery(
 		conditionTemplate, err = simpleOperatorCondition(
 			field, valueIsList, " %s != ?", " %s NOT IN (%s)",
 		)
+	case filter.CompareOperatorIsLessThan:
+		conditionTemplate, err = simpleOperatorCondition(
+			field, valueIsList, " %s < ?", " %s < LEAST(%s)",
```

Comment:
in all other cases with a list of values `filter.RequestField.Value` we chain them by OR, so I think it would be good to be consistent here. So it should be `GREATEST` here and the other way round for `CompareOperatorIsLessThanOrEqualTo`

Reply:
Thanks. 
This has been adjusted/ Checkout [b91d841](https://github.com/greenbone/opensight-golang-libraries/pull/77/commits/b91d841c0c21998f5f1224dbd63e0c7a59896430)

Reply:
Thanks. 
It has been adjusted.